### DISCUSSION TITLE
phalcon command not found

### DISCUSSION THREAD

**Discussion by laceja** [phalcon/phalcon#250] [General]
I have installed phalcon (latest version) on ubuntu 20.04, using php 7.3 (alternatives). I've used the instrustions form the phalcon.io website.  However, after completing the installation, according to the instructions, I enter phalcon at the command line and get phalcon command not found.  How can I figure out what went wrong?

**Comment by phptux** [comment]
You installed the Phalcon Framework as PHP extension. For the command line feature you need [phalcon-devtools](https://github.com/phalcon/phalcon-devtools).
